ScnSoft-Education/LeetCodeTraining.Tests/StringToInteger8.Tests.cs

```diff
@@ -0,0 +1,18 @@
+﻿using LeetCodeTraining.StringToInteger8;
+namespace LeetCodeTraining.Tests
+{
+    public class MyAtoiTests
+    {
+        [Theory]
+        [InlineData("42", 42)]
+        [InlineData("   -42", -42)]
+        [InlineData("4193 with words", 4193)]
+        [InlineData("words and 987", 0)]
```

Comment:
add unit test for:
If the integer is out of the 32-bit signed integer range [-231, 231 - 1], then clamp the integer so that it remains in the range. Specifically, integers less than -231 should be clamped to -231, and integers greater than 231 - 1 should be clamped to 231 - 1.

Reply:
added unit tests